package.json

```diff
@@ -10,10 +10,10 @@
     "test": "jest"
   },
   "dependencies": {
-    "@novnc/novnc": "^1.0.0",
-    "babel-cli": "^6.26.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "webrtc-adapter": "^6.1.5"
+    "@novnc/novnc": "1.0.0",
```

Comment:
came from the discussion here: https://github.com/rainforestapp/beak/pull/836#discussion_r223670591